go.mod

```diff
@@ -121,6 +121,7 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/dgraph-io/ristretto v0.1.1 // indirect
 	github.com/dnephin/pflag v1.0.7 // indirect
+	github.com/dropbox/dropbox-sdk-go-unofficial/v6 v6.0.5 // indirect
```

Comment:
You should run `go mod tidy`, since this import shouldn't be `indirect`, as you are using it directly.